### DISCUSSION TITLE
use case litstudy - query db

### DISCUSSION THREAD

**Discussion by edzob** [NLeSC/litstudy#48] [General]
Hi all,

just new to litstudy and to the domain of bibliometrix
the following question is a double check if I understood it correctly.

[1] I can not use litstudy to query the API of for example semanti scholar, correct?

[2] I can use litstudy to filter and complete data I downloaded from semenatic scholar, correct?

thanks for the help in understanding, best regads edzo

**Comment by stijnh** [comment]
Hi!

> [1] I can not use litstudy to query the API of for example semanti scholar, correct?

litstudy supports several data sources. Some are online API's (such as SemanticScholar) others are offline (such as CSV or bibtex files). For an overview, see [this](https://nlesc.github.io/litstudy/api/sources.html). 

For example, to search SemanticScholar using litstudy, you use:

```
import litstudy
docs = litstudy.search_semanticscholar("distributed deep learning") 
```


> [2] I can use litstudy to filter and complete data I downloaded from semenatic scholar, correct?

Functions, such as `search_semanticscholar` return a [`DocumentSet`](https://nlesc.github.io/litstudy/api/types.html#litstudy.types.DocumentSet) object. This object supports many methods like filter, union, select, sample, etc. You find an overview of all the methods [here](https://nlesc.github.io/litstudy/api/types.html#litstudy.types.DocumentSet). This allows you to select the results that you are interested in. 


Hope this helps!
